apps/chromium-browser/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  chromium:
+    image: lscr.io/linuxserver/chromium:stable
+    container_name: chromium
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Etc/UTC
+      - CHROME_CLI=https://www.linuxserver.io/ #optional
+    volumes:
+      - /path/to/config:/config
+    ports:
+      - 3000:3000
+      - 3001:3001
```

Comment:
This port can be removed, as a HTTPS-GUI with valid certificates is not possible at the moment
```suggestion
```